waardepapieren-service/configuration/waardepapieren-config-compose-travis.json

```diff
@@ -18,7 +18,7 @@
   "PRODUCT_PURPOSE" : "Bewijs verblijfadres in woonplaats",
   "SOURCE_DATA_SELECTION" : [
     {"Burgerservicenummer (BSN)" : "_embedded.ingeschrevenpersonen[0].burgerservicenummer"},
-    {"Naam" : "_embedded.ingeschrevenpersonen[0].naam"},
+    {"Voorletters" : "_embedded.ingeschrevenpersonen[0].naam.voorletters"},
```

Comment:
Do we have a last name we can use here? (Same remark for the other configs) That's probably a better sample, data-wise. Or maybe have both Voorletters and last name.

Reply:
There is no last name remarkably enough :thinking: Currently  the name object contains:
```json
"naam": {
    "aanhef": "Geachte mevrouw 'S Streeveld-te Schaars",
    "aanschrijfwijze": "jonkvrouw Ż.Å.Đ. 'S Streeveld-te Schaars",
    "gebruikInLopendeTekst": "jonkvrouw 'S Streeveld-te Schaars",
    "aanduidingNaamgebruik": "eigen_partner",
    "voornamen": "Żáïŀëñøŕ Åłéèç Đëļŧå",
    "voorletters": "Ż.Å.Đ.",
    "geslachtsnaam": "Streeveld",
    "voorvoegsel": "'S"
}
```
We can do both voorletters and aanhef?

Reply:
Geslachtsnaam is last name

Reply:
That was a unknown synonym to me. I added it to the test data of the configuration files